R/gnps.R

```diff
@@ -0,0 +1,167 @@
+#' @title GNPS spectrum similarity scores
+#'
+#' @description
+#'
+#' The similarity score used in [GNPS](https://gnps.ucsd.edu/) considers, in
+#' addition to directly matching peaks between two spectra, also matches of
+#' peaks with differences in their m/z values that correspond to the difference
+#' of the spectras' precursor m/z. For peaks that match multiple peaks in the
+#' other spectrum only the matching peak pair with the higher value/similarity
+#' is considered in the final similarity score. Peak matching that considers
+#' also differences in precursor m/z can be performed with `join_gnps` and the
+#' similarity score on such *aligned* peak matrices can be performed with
+#' `gnps`. Note that only if the two functions are used together, GNPS
+#' similarity scores are calculated.
```

Comment:
Note that GNPS similarity scores are calculated only if the two functions are used together.